packages/replay-internal/src/replay.ts

```diff
@@ -1157,8 +1161,8 @@ export class ReplayContainer implements ReplayContainerInterface {
         segmentId,
         eventContext,
         session: this.session,
-        options: this.getOptions(),
         timestamp,
+        onError: err => this.handleException(err),
```

Comment:
hm is this not a loop? We set this to call handleException, which calls onError? 🤔

Reply:
ah wait, I guess I was confused, this is a different onError here 😅